File: backend/app/models/board.rb
class Board < ApplicationRecord
  has_many :posts
  
  validates :name, uniqueness: true

  def slug 
    self.name.gsub(" ", "-")
  end

  def self.find_by_slug(slug)
    self.all.find {|game| game.slug == slug}
  end

end

File: frontend/src/redux/components/board.js
export default function Board({ board }) {

  const renderDescription = () => {
    if (board.description) {
      return <p>{board.description}</p>
    }
  }

  return (
    <div>
      <p>{board.name}</p>
      {renderDescription()}
    </div>
  )
}